services/ui/ws/display.cc

```diff
@@ -55,9 +55,11 @@ Display::~Display() {
   } else if (!window_manager_display_root_map_.empty()) {
     // If there is a |binding_| then the tree was created specifically for this
     // display (which corresponds to a WindowTreeHost).
-    window_server_->DestroyTree(window_manager_display_root_map_.begin()
-                                    ->second->window_manager_state()
-                                    ->window_tree());
+    WindowManagerDisplayRoot* display_root =
+        window_manager_display_root_map_.begin()->second;
+    if (display_root->window_manager_state())
```

Comment:
lgtm, but I think we can make this CL  a "fixup!" of the original "mojo/c++ changes..." CL, 
so that if I successfully get WindowManagerState initialized for us, these null-checks can all be removed?